docs/en/components.md

```diff
@@ -0,0 +1,42 @@
+# Components
+
+Components are standalone `.html` files which can be imported and usage in another files.
+Each component can contain own styles and scripts. Component can take some attributes and can have default values.
```

Comment:
`Every component`

`Component can receive attributes and have default values for them`